res/demo/configs/non-directory/app-blotter-nodir.json

```diff
@@ -0,0 +1,20 @@
+{
+    "devtools_port": 9090,
+    "licenseKey": "64605fac-add3-48a0-8710-64b38e96a2dd",
+    "startup_app": {
+        "name": "FDC3 Blotter POC [non-directory]",
```

Comment:
Suggest use title case a non-square brakets since this is ostensibly user-facing, so "FDC3 Blotter POC (Non-Directory)"